Mathlib/Computability/Encoding.lean

```diff
@@ -74,11 +74,12 @@ def sectionΓ'Bool : Γ' → Bool
   | Γ'.bit b => b
   | _ => Inhabited.default
 
-theorem leftInverse_section_inclusion : Function.LeftInverse sectionΓ'Bool inclusionBoolΓ' :=
```

Comment:
Can you add a deprecation warning, please?

Reply:
Done.